src/definitions/modules/calendar.js

```diff
@@ -1324,14 +1328,14 @@ $.fn.calendar.settings = {
       if (!isTimeOnly) {
         //textual month
         for (i = 0; i < words.length; i++) {
-          var word = words[i];
+          word = words[i];
```

Comment:
It seems that `var` is necessary as same in https://github.com/fomantic/Fomantic-UI/pull/1028/files#r326366079

Reply:
See above, i moved all of the declarations to the top of the function 😉